evidence.go

```diff
@@ -255,10 +268,23 @@ func (e *CcaEvidence) GetImplementationID() *[]byte {
 
 // GetRealmPublicKey returns the RMM Public Key
 // RMM Public Key is used to verify the signature on the Realm Token
-func (e *CcaEvidence) GetRealmPublicKey() *[]byte {
+func (e *Evidence) GetRealmPublicKey() *[]byte {
 	pubKey, err := e.CcaRealmClaims.GetPubKey()
 	if err != nil {
 		return nil
 	}
 	return &pubKey
 }
+
+func ecdsaPublicKeyFromRaw(data []byte) (*ecdsa.PublicKey, error) {
```

Comment:
Perhaps the best place to  put this function is in a common library we discussed earlier.

There is nothing specific to Evidence ( or Cca Evidence) for that matter!

Reply:
sure why not

Reply:
done in [72c217d](https://github.com/veraison/ccatoken/pull/10/commits/72c217d6f261b82600cb4535ffcc6479fd50868f)